docs/usage.md

```diff
@@ -195,6 +195,73 @@ $ make html
 ```
 ````
 
+### Customisation
+
+```{versionadded} 2024.12.08
+
+```
+
+Synchronisation can be disable by adding the `:no-sync:` option to a tab. It can
+also be disabled from the config file.
+
+```py
+# disable sync globally
+tabs_default_sync_behavior = "none"  # default "tab-title"
+
+# disable sync globally for labels "One" and "Two"
+tabs_no_sync_labels = {"One", "Two"}  # default set()
+```
+
+The sync label can be overwritten with the `:sync: <label-name>` option.
+
+```rst
+.. tab:: One
+   :no-sync:
+
+   Sync is disabled for this one.
+
+.. tab:: Two
+   :sync: Five
+
+   Sync with label `Five`.
+
+.. tab:: Three
+
+   Three is an odd prime.
+
+.. tab:: Four
+
+   Four is not a perfect number.
+```
+
+```{tab} One
+:no-sync:
+Sync is disabled for this one.
+```
+
+```{tab} Two
+:sync: Five
+Sync with label `Five`.
+```
+
+```{tab} Three
+Three is an odd prime.
+```
+
+```{tab} Four
+Four is not a perfect number.
+```
+
+<!--  -->
```

Comment:
This looks like a redundant comment.

Reply:
No, this is intentional to force the creation of a new tab group.
Added a comment to make it more obvious.

<img width="785" alt="Screenshot 2025-03-09 at 20 15 47" src="https://github.com/user-attachments/assets/6de84ab1-ec87-49c0-a576-c0a06edbf9ee" />